packages/next/src/lib/verify-typescript-setup.ts

```diff
@@ -129,7 +131,7 @@ export async function verifyTypeScriptSetup({
     // Next.js' types:
     await writeAppTypeDeclarations({
       baseDir: dir,
-      imageImportsEnabled: !disableStaticImages,
+      imageImportsEnabled: !disableStaticImages && !customImageTypes,
```

Comment:
What is the difference between `customImageTypes` and `disableStaticImages`. Why not use the existing `disableStaticImages` instead?